src/providers/ad/ad_gpo.c

```diff
@@ -930,39 +1080,39 @@ ad_gpo_filter_gpos_by_dacl(TALLOC_CTX *mem_ctx,
 
         /* gpo_flags value of 2 means that GPO's computer portion is disabled */
         if (candidate_gpo->gpo_flags == 2) {
-            DEBUG(SSSDBG_TRACE_ALL,
+            DEBUG(SSSDBG_TRACE_FUNC,
                   "GPO not applicable to target per security filtering: "
                   "GPO's computer portion is disabled\n");
             continue;
         }
 
-        /*
-         * [MS-ADTS] 5.1.3.3.4:
-         * If the security descriptor has no DACL or its "DACL Present" bit
-         * is not set, then grant requester the requested control access right.
-         */
+        if (((sd->type & SEC_DESC_DACL_PRESENT)) && (dacl != NULL)) {
```

Comment:
Just a nitpick, the double braces in `((sd->type & SEC_DESC_DACL_PRESENT))` are not needed, a single pair should be sufficient.

Reply:
Sure, stupid failure.